lib/cell.ex

```diff
@@ -0,0 +1,18 @@
+defmodule GameOfLife.Cell do
+  def live(cell_map, location) do 
+      %{cell_map | location => :alive}  
+  end   
+
+  def die(cell_map, location) do 
+      %{cell_map | location => :dead}  
+  end   
+
+  def is_alive?(cells, location) do 
```

Comment:
This is confusing. Your first argument is a single cell, but the method names the parameter as plural "cells".